libs/state/actions/src/lib/actions.factory.ts

```diff
@@ -96,12 +105,4 @@ export class RxActionFactory<T extends Partial<Actions>> implements OnDestroy {
       Object.values(s).forEach((subject: any) => subject.complete());
     });
   }
-
-  /**
-   * @internal
-   * Internally used to clean up potential subscriptions to the subjects. (For Actions it is most probably a rare case but still important to care about)
-   */
-  ngOnDestroy() {
```

Comment:
Is it ok to remove this method without providing a migration? It could be a breaking change if somebody uses this method. However it is marked as internal so nobody should use it

Reply:
> However it is marked as internal so nobody should use it

+1 I'm for removing it completely

Reply:
go for it